### PR TITLE
Actiontag gtm

### PR DESCRIPTION
**Pull request by ryanmoorman**
Made some changes to the values tracked for GTM/GA Enhanced E-Commerce.

### CODE REVIEW

File: docs/action-tag/examples/overview.jade [hunk 1]
```diff
@@ -92,6 +92,29 @@ block description
     Note, for this case, if Spouse is not in the original form definition, the API would not be able to parse it, so you will need to intercept and store that POST value somewhere else.
 
     Also note that since the added definition has a queryString of `'sp'`, it will fill if the page has a query string argument like: `?sp=Joe`
+
+  h4 Ecommerce tracking in Google Tag Manager
+  :markdown
+    Google Tag Manager (GTM) is one of the best tools for grouping tags of various types into a unified script that can be executed on pageload. GTM is primarily used to manage Google Analytics (GA) tags. 
+
+    In ActionTag, we use GTM/GA to track product metrics for our own internal reporting metrics. Because of the way we have configured how some events work in relation to GTM, it is easy to utilize our implementation for your own organization's GTM/GA implementation.
```
Comment (shaisachs) [inline docs/action-tag/examples/overview.jade]: Just a little wordsmithing, but this should be "for our own internal reporting purposes".  And that second sentence is a little awkward, maybe: "We've configured GA events to be easy to use for your own organization’s GTM/GA implementation."

File: docs/action-tag/examples/overview.jade [hunk 2]
```diff
@@ -92,6 +92,29 @@ block description
...
+
+    We took the time to build in events that are fired on successful form submits for Contribution and Event form types so information can be easily tracked as ECommerce data in your own GTM/GA setup.
```
Comment (shaisachs) [inline docs/action-tag/examples/overview.jade]: A bit wordy.  Replace with "Events are fired on successful form submits for Contribution and Event form types, so information can be easily tracked as ecommerce data."

File: docs/action-tag/examples/overview.jade [hunk 3]
```diff
@@ -92,6 +92,29 @@ block description
...
+      'transactionTotal': 38.26,  // Transaction Total
+      'transactionProducts': [{
+          'category': 'ContributionForm', //Page Type
+          'sku': '1', // SKU/code based upon session ID
+          'price': 11.99, // Unit Price (Transaction Amount)
+          'quantity': 1, // Quantity
+          'brand': 'People for Good VA' // NGP Tenant (Account) Name
+      }]
+    }];
+    ````
```
Comment (shaisachs) [inline docs/action-tag/examples/overview.jade]: The example above is I think lacking some context, can you provide some more detail on what is actually going on?  For example, "The following is an example of a successful contribution submission event pushed to GA."

Finally, since events have additional data added for ticketing, we should probably document what that data is.

Also too, this documentation is really part of the ActionTag API, and not an "example"... is it?  We should put it in a new section, "Google Analytics Integration" perhaps, after "Callbacks".

**Comment by ryanmoorman** [comment]
Closing request and opening a new one with changes mentioned in the comments of this pull request.
